### DISCUSSION TITLE
Example of using Dagster with DVC ?

### DISCUSSION THREAD

**Discussion by jomeier** [dagster-io/dagster#19056] [Community]
Hi everybody,

I would be glad if someone could provide an example of how it looks like, when Dagster is integrated with **DVC** (for version control of data). 

Would DVC be used together with IOManagers?

Any examples or ideas are welcome.

Thanks and greetings,

Josef

**Comment by Alex-Kopylov** [comment]
+1 That would be great!

Dagster have some similar mechanisms as DVC under the hood [Asset versioning and caching](https://docs.dagster.io/guides/dagster/asset-versioning-and-caching)

It would be nice to have tutorials and examples of end-to-end ML project that demonstrates reproducibility of experiments. How to easily checkout to data-code versions combinations, manage experiments, etc.

DVC is good at it. But it's not so versatile when it comes to data pipelines, in comparison with Dagster.
